src/test/java/com/epam/tc/hw2/ex1/LoginPageTest.java

```diff
@@ -0,0 +1,120 @@
+        WebElement homeOnTheHeader = webDriver.findElement(By.xpath("//a[@href='index.html']"));
+        homeOnTheHeader.isDisplayed();
+        WebElement contactsOnTheHeader = webDriver.findElement(By.xpath("//a[@href='contacts.html']"));
+        contactsOnTheHeader.isDisplayed();
+        WebElement serviceOnTheHeader = webDriver.findElement(By.cssSelector(".dropdown"));
+        serviceOnTheHeader.isDisplayed();
+        WebElement metalsColors = webDriver.findElement(By.xpath("//a[@href='metals-colors.html']"));
+        metalsColors.isDisplayed();
+        softly.assertThat(homeOnTheHeader.getText()).isEqualTo("HOME");
+        softly.assertThat(contactsOnTheHeader.getText()).isEqualTo("CONTACT FORM");
+        softly.assertThat(serviceOnTheHeader.getText()).isEqualTo("SERVICE");
+        softly.assertThat(metalsColors.getText()).isEqualTo("METALS & COLORS");
+        //Assert that there are 4 images on the Index Page and they are displayed
+        webDriver.findElement(By.xpath("//img[contains(@id,'epam-logo')]")).isDisplayed();
+        webDriver.findElement(By.xpath("//img[contains(@id,'user-icon')]")).isDisplayed();
+        WebElement icon1 = webDriver.findElement(By.className("icon-practise"));
+        icon1.isDisplayed();
+        WebElement icon2 = webDriver.findElement(By.className("icon-custom"));
+        icon2.isDisplayed();
+        WebElement icon3 = webDriver.findElement(By.className("icon-multi"));
+        icon3.isDisplayed();
+        WebElement icon4 = webDriver.findElement(By.className("icon-base"));
+        icon4.isDisplayed();
+        //Assert that there are 4 texts on the Index Page under icons and they have proper text
+        WebElement textUnder1 =
+            webDriver.findElement(By.xpath("//div[@class='col-sm-3']//*[contains(text(),'To include')]"));
+        textUnder1.isDisplayed();
+        softly.assertThat(textUnder1.getText()).isEqualTo("To include good practices\n"
+            + "and ideas from successful\n"
+            + "EPAM project");
+        WebElement textUnder2 =
+            webDriver.findElement(By.xpath("//div[@class='col-sm-3']//*[contains(text(),'To be flexible')]"));
+        textUnder2.isDisplayed();
+        softly.assertThat(textUnder2.getText()).isEqualTo("To be flexible and\n"
+            + "customizable");
+        WebElement textUnder3 =
+            webDriver.findElement(By.xpath("//div[@class='col-sm-3']//*[contains(text(),'To be multiplatform')]"));
+        textUnder3.isDisplayed();
+        softly.assertThat(textUnder3.getText()).isEqualTo("To be multiplatform");
+        WebElement textUnder4 =
+            webDriver.findElement(By.xpath("//div[@class='col-sm-3']//*[contains(text(),'Already have good base')]"));
+        textUnder4.isDisplayed();
+        softly.assertThat(textUnder4.getText()).isEqualTo("Already have good base\n"
+            + "(about 20 internal and\n"
+            + "some external projects),\n"
+            + "wish to get more…");
```

Comment:
All web elements for the menu could be getter as `List<WebElement>` and assertion could be used through collection

Reply:
Looks like this comment was ignored